analyzers/src/ITs.JsonParser/Json/SarifIssues.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace ITs.JsonParser.Json;
+
+// Format of JSONs in "output/"
+internal class SarifIssues
+{
+    public Run[] Runs { get; set; }
+
+    public SarifIssue[] AllIssues() => Runs.SelectMany(r => r.Results).ToArray();
+}
+
+internal class Run
+{
+    public SarifIssue[] Results { get; set; }
+}
+
+internal class SarifIssue
+{
+    public string RuleId { get; set; }
+    public string Level { get; set; }
+    public string Message { get; set; }
+    public LocationWrapper[] Locations { get; set; }
+
+    public Location Location => Locations is { Length: > 0 } ? Locations[0].ResultFile : null;
+
+    public object Order() =>
+        Location is not null
+            ? (NormalizedUri(), Location.Region.StartLine, Location.Region.StartColumn, Location.Region.EndLine, Location.Region.EndColumn, Message)
+            : default;
+
+    public string NormalizedUri()
+    {
+        if (Location is not null)
+        {
+            const string prefix = "https://github.com/SonarSource/sonar-dotnet/blob/master/";
+            // analyzers/its/sources/project/...
+            var filepath = Location.Uri[Location.Uri.IndexOf("analyzers/its")..];
```

Comment:
It was suggested by the IDE, and I wanted to see what you think about it :)
I agree, it feels weird, but I am not sure if it is because we are not used to this or if it is indeed cryptic.
I'll revert it to SubString for now.

Reply:
#8683

All of the `..` and `^` are very cryptic, especially around the end indexes.